hub/api/tests/test_dataset_utils.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from hub.api.dataset_utils import get_compressor
+from hub.util.exceptions import InvalidCompressor
+
+
+@pytest.mark.parametrize("codec_name", (mod for mod in dir() if mod.isupper()))
```

Comment:
why is mod.isupper() being called? also instead of doing this inline, it should be pre-created in a variable, something like `CODECS`.